OpenLIFULib/OpenLIFULib/algorithm_input_widget.py

```diff
@@ -143,6 +162,16 @@ def _populate_from_session(self) -> None:
 
         self.set_session_related_combobox_tooltip("This choice is fixed by the active session")
 
+        # Update photoscan combo box
+        if "Photoscan" in self.inputs_dict:
+            if affiliated_photoscans_list is None:
```

Comment:
as mentioned in other comments, it's a bit weird that `None` represents the list being empty. it can just _be_ empty, and we can check here

```suggestion
            if len(affiliated_photoscans_list)==0:
```